python/across_api/base/pointing.py

```diff
@@ -0,0 +1,20 @@
+# Copyright © 2023 United States Government as represented by the
+# Administrator of the National Aeronautics and Space Administration.
+# All Rights Reserved.
+
+
+class PointingBase:
+    """
+    Defined class to represent an observation of an instrument
+
+    TODO: Minimally defined for sake of BurstCube MVP
+    """
+
+    ra: float
+    dec: float
+    position_angle: float
+
+    def __init__(self, ra, dec, position_angle) -> None:
+        self.ra = ra
+        self.dec = dec
+        self.position_angle = position_angle
```

Comment:
I suggest removing this class and replacing it with [SkyOffsetFrame](https://docs.astropy.org/en/stable/api/astropy.coordinates.SkyOffsetFrame.html#astropy.coordinates.SkyOffsetFrame) which consists of an Astropy coordinate origin and a position angle.

Reply:
I want to hold off on abstracting this to other existing classes. I am hoping/foreseeing this class becoming more robust when we want to implement more functionality when it comes to observational analysis. 

Things like `pointing_type` being `photometric` or `spectroscopic`. Other fields like `exposure_time` and `obsdate` being taken into account as well. 

This class won't be just for positional information.

If you have ideas with that scope, then I'd be down to discuss those in respect to this class.